prog/vm/github_runner.rb

```diff
@@ -188,6 +188,9 @@ def before_run
   end
 
   label def register_runner
+    # Do not register runner if the runner-script service is already running.
+    hop_wait unless vm.sshable.cmd("systemctl show -p SubState --value #{SERVICE_NAME}").chomp == "dead"
```

Comment:
This handles the case where we execute the vm.sshable.cmd but not hop_wait.

How about the case where we execute github_client.post but not the rest. Would we get "Already exist" error and then delete the runner? We shouldn't delete the runner in that case, no?